src/traces/parcoords/attributes.js

```diff
@@ -0,0 +1,133 @@
+            ],
+            dflt: [0, 1],
+            description: [
+                'Sets the vertical domain of this `parcoords` trace',
+                '(in plot fraction).'
+            ].join(' ')
+        }
+    },
+
+    dimensions: {
+        _isLinkedToArray: 'dimension',
+        label: {
+            valType: 'string',
+            role: 'info',
+            description: 'The shown name of the dimension.'
+        },
+        tickvals: axesAttrs.tickvals,
+        ticktext: axesAttrs.ticktext,
+        visible: {
+            valType: 'boolean',
+            dflt: true,
+            role: 'info',
+            description: 'Shows the dimension when set to `true` (the default). Hides the dimension for `false`.'
+        },
+        range: {
+            valType: 'data_array',
+            role: 'info',
+            description: [
+                'The domain range that represents the full, shown axis extent. Defaults to the `values` extent.',
+                'Must be an array of `[fromValue, toValue]` with finite numbers as elements.'
+            ].join(' ')
+        },
+        constraintrange: {
+            valType: 'data_array',
+            role: 'info',
+            description: [
+                'The domain range to which the filter on the dimension is constrained. Must be an array',
+                'of `[fromValue, toValue]` with finite numbers as elements.'
+            ].join(' ')
+        },
+        values: {
+            valType: 'data_array',
+            role: 'info',
+            description: [
+                'Dimension values. `values[n]` represents the value of the `n`th point in the dataset,',
+                'therefore the `values` vector for all dimensions must be the same (longer vectors',
+                'will be truncated). Each value must be a finite number.'
+            ].join(' ')
+        },
+        description: 'The dimensions (variables) of the parallel coordinates chart. 2..60 dimensions are supported.'
+    },
+
+    pad: extendDeep({}, padAttrs, {
+        description: 'Set the padding of the `parcoords` component along each side to provide space for annotations.'
+    }, {t: {dflt: 80}, l: {dflt: 80}, r: {dflt: 80}, b: {dflt: 80}}),
+
+    line: extendFlat({},
+
+        // the default autocolorscale isn't quite usable for parcoords due to context ambiguity around 0 (grey, off-white)
+        // autocolorscale therefore defaults to false too, to avoid being overridden by the  blue-white-red autocolor palette
+        extendDeep(
+            {},
+            colorAttributes('line'),
+            {
+                colorscale: extendDeep(
+                    {},
+                    colorAttributes('line').colorscale,
+                    {dflt: colorscales.Viridis}
+                ),
+                autocolorscale: extendDeep(
+                    {},
+                    colorAttributes('line').autocolorscale,
+                    {dflt: false}
```

Comment:
Yes, the current plotly default palette is often a bad choice for parcoords, though there may be specific datasets and styling needs where it's sensible. I saw that for certain things (see link below), `autocolorscale` defaults to false too, so I left it as it is and I updated the description, your call is definitely warranted. If, as @etpinard suggested, a future version of Plotly.js would use Viridis rather than the current red/blue palette as a default, `parcoords.autocolorscale` could be reverted to true again.

If there's a decision for getting rid of `autocolorscale`, I can take a stab at it though I'm a bit pressed for time. It looks a bit delicate as this attribute is intertwined in the current code that's in effect for all other traces too, e.g. [below this](https://github.com/plotly/plotly.js/blob/master/src/components/colorscale/defaults.js#L41) so not having `autocolorscale` would be a new thing to Plotly. Let me know if I should proceed with it though.